src/components/asthma/helpers/daily-data-providers/air-quality-home.ts

```diff
@@ -0,0 +1,6 @@
+﻿import { DailyDataQueryResult } from '../../../../helpers/query-daily-data';
+import { collateMaxValueDataPoints, queryAsthmaDeviceData } from './shared';
+
+export default function (startDate: Date, endDate: Date): Promise<DailyDataQueryResult> {
+    return queryAsthmaDeviceData('AirNowApi', 'HomeAirQuality', startDate, endDate, collateMaxValueDataPoints);
```

Comment:
Can this be decoupled from Asthma? For other studies where AirQuality may be relevant

Reply:
Yep, I have moved this into the common daily data providers area.